R/quanteda_options.R

```diff
@@ -138,6 +138,9 @@ get_options_default <- function(){
                  verbose = FALSE,
                  print_dfm_max_ndoc = 20L,
                  print_dfm_max_nfeat = 20L,
+                 print_corpus_max_ndoc = 0L,
+                 print_corpus_max_nchar = 0L,
```

Comment:
I am `print_corpus_max_ndoc = 0L` is fine but think that `print_corpus_max_nchar = 100L` is better, because users see only dots by default.
```
> print(data_corpus_inaugural, ndoc = 10)
Corpus consisting of 58 documents and 3 docvars.
 [1789-Washington]  ...
 [1793-Washington]  ...
      [1797-Adams]  ...
  [1801-Jefferson]  ...
  [1805-Jefferson]  ...
    [1809-Madison]  ...
    [1813-Madison]  ...
     [1817-Monroe]  ...
     [1821-Monroe]  ...
      [1825-Adams]  ...
```

Reply:
Agreed. I changed the default to 60L, to allow for long filenames but still to keep it within a relatively small line width.